pkg/volume/csi/csi_mounter.go

```diff
@@ -233,11 +233,14 @@ func (c *csiMountMgr) podAttributes() (map[string]string, error) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
 		return nil, nil
 	}
-	if c.plugin.csiDriverLister == nil {
+	//if c.plugin.csiDriverLister == nil {
+	csiDriverLister := c.plugin.host.GetCSIDriverLister()
```

Comment:
if csiDriverLister is feature gated this will become a common-ish case. Is returning an error the right thing to do here?

Reply:
Probably not needed.  I removed the check.

Reply:
is it possible this is nil?

Reply:
yes, when feature gates are not set the lister will not be initialized